### ISSUE TITLE
cannot load Kolibri on Firefox/Android and Safari/Mac

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#1919]
Testing in browserstack on a number of Android platforms, Kolibri fails to load:

![image](https://user-images.githubusercontent.com/2367265/28734678-f5ce019c-7396-11e7-92e0-b6971f7ccc99.png)

**Comment by christianmemije** [comment]
The stacktrace:
```
ReferenceError: Intl is not defined[Learn More]  default_frontend-0.5.0beta4.js:196:4763
TypeError: kolibriGlobal is undefined[Learn More]  learn:56:1
TypeError: kolibriGlobal is undefined[Learn More]  learn:71:18
TypeError: kolibriGlobal is undefined[Learn More]  learn:75:5
TypeError: kolibriGlobal is undefined[Learn More]  learn:79:1
TypeError: kolibriGlobal is undefined[Learn More]  learn:80:1
TypeError: kolibriGlobal is undefined[Learn More]  learn:81:1
TypeError: kolibriGlobal is undefined[Learn More]  learn:82:1
TypeError: kolibriGlobal is undefined[Learn More]  learn:88:5
TypeError: kolibriGlobal is undefined[Learn More]  learn:90:5
TypeError: kolibriGlobal is undefined[Learn More]  learn:92:5
TypeError: kolibriGlobal is undefined[Learn More]  learn:94:5
TypeError: kolibriGlobal is undefined[Learn More]  learn_module-0.5.0beta4.js:1:7847
```

**Comment by rtibbles** [comment]
Looks like our Intl polyfill is failing.

**Comment by rtibbles** [comment]
I would kind of expect Firefox to have implemented the Intl API, they're normally so keen on standards compliance.

**Comment by ivanistheone** [comment]
Similar problem in Safari on Mac OS:

<img width="905" alt="screen shot 2017-08-22 at 11 34 22 am" src="https://user-images.githubusercontent.com/163966/29573855-02845972-872e-11e7-9185-8eb896cc1496.png">


Note Safari problem is also in 0.5.beta3, not just 0.6:

<img width="941" alt="screen shot 2017-08-22 at 11 36 47 am" src="https://user-images.githubusercontent.com/163966/29573962-3f0bce84-872e-11e7-86d0-54a34e9ac1d5.png">

**Comment by christianmemije** [comment]
Fixed in #2012
